Dockerfile

```diff
@@ -12,11 +12,14 @@ LABEL description="Mintmaker - Renovate custom image" \
       url="https://github.com/konflux-ci/mintmaker-renovate-image/" \
       vendor="Red Hat, Inc."
 
-ARG RENOVATE_VERSION=37.413.2-custom
+ARG RENOVATE_VERSION=38.55.2-rpm
+
+ARG NODEJS_VERSION=20.17.0
```

Comment:
```suggestion
ARG RENOVATE_VERSION=38.55.2-rpm

# NodeJS version used for Renovate, has to satisfy the version
# specified in Renovate's package.json
ARG NODEJS_VERSION=20.17.0
```

I'd just put this comment here to warn against just updating this randomly :smile: